src/modules/header/styles.css

```diff
@@ -285,7 +311,7 @@
   --form-control-color: var(--gray-700);
   max-width: 100%;
 
-  padding: calc(.25rem - 1px) .25rem .25rem;
+  padding: calc(.25rem - 1px) 0 .25rem;
```

Comment:
No right border looks a bit weird on the design preview but I hope it's fine on the full-width website.

Reply:
The border is there. The padding is removed from left and right so all calculations are easier and also it aligns with result card correctly.

Reply:
I see what you meant by your comment. This is just an issue with the preview. It shouldn't happen in production